### PR TITLE
Utilize isNotEqualTo

### PR DESCRIPTION
**Pull request by Neviothr**
**When merged this pull request will:**
- title

### CODE REVIEW

**Comment by PabstMirror** [comment]
looks good, but I don't think `isNotEqualTo` will make it into 2.00, so it may be a while until this can be merged

File: addons/settings/fnc_export.sqf [hunk 1]
```diff
@@ -43,7 +43,7 @@ private _temp = [];
     };
 
     if (_exportDefault || {
-        !(_value isEqualTo _defaultValue) ||
+        (_value isNotEqualTo _defaultValue) ||
```
Comment (commy2) [inline addons/settings/fnc_export.sqf:46]: You can get rid of these parentheses

**Review by commy2 — APPROVED** [review]

**Comment by commy2** [comment]
do not merge before isNotEqualTo is in

**Review by PabstMirror — APPROVED** [review]
lgtm

**Pull request merged**